pipelines/tekton/aiedge-e2e/examples/tensorflow-housing.trigger.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: Trigger
+metadata:
+  name: tensorflow-housing
+spec:
+  bindings:
+  - name: git-containerfile-revision
+    value: $(body.head_commit.id)
+  - name: git-containerfile-repo
+    value: $(body.repository.clone_url)
```

Comment:
The value here is a field in the payload that was POSTed by GitHub to the webhook endpoint, the name maps it to a pipeline parameter.